canonical-web-and-design/ubuntu.com
# Copy Update on /aws/pro

## Done

- Update What's in Ubuntu Pro matrix as per copy doc 

## QA

- View site at: https://ubuntu-com-9211.demos.haus/aws/pro
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check against [copy doc](https://docs.google.com/document/d/1pLRWJZSpE04nYt-yHBtCjxGI7VZsigcP2UJIBmR8i-0/edit) and [issue](https://github.com/canonical-web-and-design/ubuntu.com/issues/9210) 


## Issue / Card

Fixes #9210 

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/107209962-6bba2080-69fb-11eb-8320-c1f77426d4df.png)
